project.clj

```diff
@@ -2,14 +2,17 @@
   :description "FIXME: write description"
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [clj-time "0.9.0"] ; required due to bug in lein-ring
-                 [metosin/schema-tools "0.7.0"]
+                 [prismatic/schema "1.1.0"]
```

Comment:
we may run into issues if we use a version of schema which is not supported by compojure-api